mmseg/datasets/__init__.py

```diff
@@ -9,16 +9,31 @@
 from .dataset_wrappers import ConcatDataset, RepeatDataset
 from .drive import DRIVEDataset
 from .hrf import HRFDataset
+from .loveda import LoveDADataset
 from .night_driving import NightDrivingDataset
 from .pascal_context import PascalContextDataset, PascalContextDataset59
 from .stare import STAREDataset
 from .voc import PascalVOCDataset
 
 __all__ = [
-    'CustomDataset', 'build_dataloader', 'ConcatDataset', 'RepeatDataset',
-    'DATASETS', 'build_dataset', 'PIPELINES', 'CityscapesDataset',
-    'PascalVOCDataset', 'ADE20KDataset', 'PascalContextDataset',
-    'PascalContextDataset59', 'ChaseDB1Dataset', 'DRIVEDataset', 'HRFDataset',
-    'STAREDataset', 'DarkZurichDataset', 'NightDrivingDataset',
-    'COCOStuffDataset'
+    'CustomDataset',
+    'build_dataloader',
+    'ConcatDataset',
+    'RepeatDataset',
+    'DATASETS',
+    'build_dataset',
+    'PIPELINES',
+    'CityscapesDataset',
+    'PascalVOCDataset',
+    'ADE20KDataset',
+    'PascalContextDataset',
+    'PascalContextDataset59',
+    'ChaseDB1Dataset',
+    'DRIVEDataset',
+    'HRFDataset',
+    'STAREDataset',
+    'DarkZurichDataset',
+    'NightDrivingDataset',
+    'COCOStuffDataset',
+    'LoveDADataset',
```

Comment:
```suggestion
    'LoveDADataset'
```

Reply:
@MengzhangLI What should we do? Create a new PR for what?